src/utils.c

```diff
@@ -226,7 +228,7 @@ pu_write_raw(const gchar *input_path,
 
         if (g_output_stream_write(output_ostream, buffer, num_read, NULL, error) < 0)
             return FALSE;
-        input_size -= num_read;
+        input_remaining -= num_read;
```

Comment:
Could you add an empty line between the previous if-statement and this line? I oversaw this styling issue in a previous PR adding this line. Would be nice to have it fixed here.